docs/python_sdk_v2.1.md

```diff
@@ -0,0 +1,221 @@
+# Xailient SDK API Documentation
+
+**Xailient SDK Version: 2.0**
```

Comment:
should this be 2.1.1 instead?

Reply:
Shall we rename the file to python_sdk_v2.md and keep only this version?

Reply:
i have changed as recommended